stl/inc/xmemory

```diff
@@ -2395,7 +2395,7 @@ _CONSTEXPR20 typename _Container::size_type _Erase_remove(_Container& _Cont, con
     const auto _Old_size = _Cont.size();
     _STD _Seek_wrapped(_First, _STD remove(_STD _Get_unwrapped(_First), _STD _Get_unwrapped(_Last), _Val));
     _Cont.erase(_First, _Last);
-    return _Old_size - _Cont.size();
+    return static_cast<typename _Container::size_type>(_Old_size - _Cont.size());

```

Comment:
It seems that `_Erase_remove(_if)` can be C++20 (and later) -only now, which allows us to drop `typename`, as we've removed `std::experimental::erase(_if)`.